.gitignore

```diff
@@ -0,0 +1 @@
+ServiceFinder/ServiceFinder/__pycache__/
```

Comment:
how does the directory structure work? Do we need the outer ServiceFinder directory?